.github/workflows/main-push.yml

```diff
@@ -37,14 +37,14 @@ jobs:
       shell: bash
       run: |
         cabal update
-        cabal new-build all -f Pedantic
+        cabal build all -f Pedantic
 
     - name: Upload package
       env:
         HACKAGE_PASSWORD: ${{ secrets.HACKAGE_PASSWORD }}
       shell: bash
       run: |
-        cabal sdist
+        cabal sdist all
```

Comment:
So now we have 4 package candidates on Hackage, right?

Reply:
Yes, possibly we should only upload the `symbolic-base` one.

Reply:
Changed to upload only `symbolic-base` now.